packages/terra-clinical-result/src/flowsheetresultcell/FlowsheetResultCell.module.scss

```diff
@@ -0,0 +1,178 @@
+:local {
+  .flowsheet-result-cell {
+    align-items: center;
+    display: flex;
+    margin: 0;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // Needed for IE10
+    padding: 0;
+    width: 100%;
+  }
+
+  .single-result-display {
+    align-items: center;
+    display: flex;
+    flex-grow: 1;
+    height: 1.78571em;
+    margin: 0;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // Needed for IE10
+    padding: 0;
+  }
+
+  .additional-results-display {
+    display: flex;
+    flex: 0 0 auto;
+    height: 1.78571em;
+    min-width: 0;
+
+    &.add-end-accessory-space {
+      padding-right: 1em;
+    }
+
+    /* stylelint-disable max-nesting-depth */
+    &.interpretation-neutral {
+      .additional-results-value {
+        background-color: #f3f3f3;
+        border: 1px solid #686c6e;
+        color: #6a6c6e;
+        font-weight: normal;
+
+        &:nth-of-type(2) {
+          color: #686c6e;
+        }
+      }
+    }
+      
+    &.interpretation-high {
+      .additional-results-value {
+        background-color: #ffdede;
+        border: 1px solid #f00;
+        color: #6a6c6e;
+        font-weight: bold;
+  
+        &:nth-of-type(2) {
+          color: #ffdede;
+        }
+      }
+    }
+  
+    &.interpretation-critical {
+      .additional-results-value {
+        background-color: #de0000;
+        border: 1px solid #fff;
+        color: #fff;
+        font-weight: bold;
+  
+        &:nth-of-type(2) {
+          color: #de0000;
+          left: 0.30769em; // Must use ems for font re-scaling
+          top: -1.30769em; // Must use ems for font re-scaling
+        }
+      }
+    }
+    /* stylelint-enable max-nesting-depth */
+  }
+
+  .additional-results-stack {
+    box-sizing: border-box;
+    display: block;
+    height: 100%;
+    min-width: 0;
+    padding: 0 0.28571em; // Must use ems for font re-scaling
+    position: relative;
+    width: 100%;
+  }
+
+  .additional-results-value {
+    background-color: #f3f3f3;
+    border: 1px solid #686c6e;
+    box-sizing: border-box;
+    color: #6a6c6e;
+    display: block;
+    font-size: 0.92857em; // Must use ems for font re-scaling
+    font-weight: normal;
+    //height: 1.69230em; // Must use ems for font re-scaling
+    left: 0;
+    line-height: 1.53846;
+    min-width: 1.07142em; // Must use ems for font re-scaling
+    overflow: hidden;
+    padding: 0 0.21428em; // Must use ems for font re-scaling
+    position: relative;
+    top: 0;
+    z-index: 2;
+
+    &:nth-of-type(2) {
+      color: #686c6e;
+      left: 0.23076em; // Must use ems for font re-scaling
+      top: -1.38461em; // Must use ems for font re-scaling
+      z-index: 1;
+    }
+
+    &.additional-results-max-value {
+      padding: 0 0.07142em 0 0.14285em; // Must use ems for font re-scaling
+    }
+  }
+
+  .end-accessory-icons {
+    display: flex;
+    flex: 0 0 auto;
+    height: 1.78571em; //1.71428em; // Must use ems for font re-scaling
```

Comment:
some clean up in the comments

Reply:
Cleaned https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-2d062ffcb33cf769fd482a8487416461R121